pycognaize/document/snapshot.py

```diff
@@ -20,6 +21,17 @@ def documents(self) -> Mapping:
         """
         return self._documents
 
+    @staticmethod
+    def download(destination_dir: str):
+        """Downloads snapshot to specified destination"""
+        login_instance = Login()
```

Comment:
Add option to specify the snapshot id and snapshot path, default to env variable if not provided.